overlord/snapstate/catalogrefresh.go

```diff
@@ -79,6 +79,10 @@ func (r *catalogRefresh) Ensure() error {
 	return err
 }
 
+func RefreshCatalogs(st *state.State) error {
```

Comment:
eh, you might as well just expose `refreshCatalogs`, and do `snapstateRefreshCatalogs(st, snapstate.Store(st))` in `daemon`